src/vaadin-grid-data-provider-mixin.html

```diff
@@ -338,22 +365,38 @@
 
     /**
      * Clears the cached pages and reloads data from dataprovider when needed.
+     * When an item is provided, will only clear - an reload - cache containing this item.
+     *
+     * @param  {Object}   item - a model to find
+     * @param  {Function} find - the find function: function(a,b) {return true when a === b}}
```

Comment:
I think the find function shouldn't be included in the API. The item instances may not equal by reference but there's already a way to match items by a primitive sub-property: the `itemIdPath` -property. You can internally use the `_itemsEqual` function for matching.

Reply:
use case is as in the above example, where I have to manage instance matching between two datastores (one for the tree grid, one for the synced view) having different key mechanisms: 
 `this.$.grid.clearCache(modelThatIsDroppedOnTheView, (a, b) => a.$key === b.key);`

Reply:
Wouldn't this work (without the matcher function)?
```
this.$.grid.clearCache({$key: modelThatIsDroppedOnTheView.key});
```

Reply:
a84194097a0c0fc52545401e22ce33c950be3d6c